app/controllers/articles_controller.rb

```diff
@@ -3,4 +3,10 @@ class ArticlesController < ApplicationController
   def index
     @articles = Article.cached_all
   end
+
+  def show
+    @article = Article.cached_find(params[:id])
+    # @comments = Article.cached_comments(params[:id])
+    @comments = Comment.cached_all(params[:id])
```

Comment:
@phendych Comment.cached_all? this will show all comments?
why pass the article's id into comments? why not do @article.cached_comments?